pkg/resources/diff_suppressions.go

```diff
@@ -185,3 +187,19 @@ func IgnoreValuesFromSetIfParamSet(key, param string, values []string) schema.Sc
 		return slices.Contains(values, old)
 	}
 }
+
+func suppressIdentifierQuoting(_, oldValue, newValue string, _ *schema.ResourceData) bool {
+	if oldValue == "" || newValue == "" {
+		return false
+	} else {
+		oldId, err := sdk.ParseIdentifierString(oldValue)
```

Comment:
Nit: this doesn't need to be in else block.

Reply:
Removed